apps/glimmer/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { ComponentManager, setPropertyDidChange } from '@glimmer/component';
+import App from './main';
+
+const app = new App();
+const containerElement = document.getElementById('app');
+
+setPropertyDidChange(() => {
+  app.scheduleRerender();
+});
+
+app.registerInitializer({
+  initialize(registry) {
+    registry.register(`component-manager:/${app.rootName}/component-managers/main`, ComponentManager);
+  }
+});
+
+app.renderComponent('Glimmer', containerElement, null);
+
+app.boot();
+
+setTimeout(()=>{
+  if (typeof API !== 'undefined') {
+    API.render = function(){ 
+      if (window.appComponent) {
+        window.appComponent.setState(API.store);
```

Comment:
This is where you should do `app._rerender();`, and then remove the app.scheduleRerender from setPropertyDidChange. We are testing raw dom reconcilation performance - how quickly can each implementation bring the view in sync with state. All other implementations are doing this synchronously after each change.

Reply:
@somebee nope, we need to call scheduleRerender to run rerender in next runloop (and it will be synchronous), because inside setState I do 2 modifications in current loop (property assigment)

Reply:
See my other comment. You obviously cannot do asynchronous rendering while the other implementations are not. The whole point is to test raw reconciler performance. As I mentioned in the comment, could you remove the `@tracked`, and just rerender without tracking the properties at all? This is the equivalent to what the other implementations are doing. I agree that it is not fair if glimmer ends up reconciling two times per step. But it went from being 10x faster to 25x slower, which I'm pretty sure is not due to synchronous rendering two times instead of one.

Reply:
it make sense